packages/strapi-plugin-open-forms-embed/admin/src/components/CustomCombobox/index.tsx

```diff
@@ -64,12 +64,14 @@ function CustomCombobox({
     if (config?.api_url && config?.token && apiUrl) {
       fetchAllOpenForms();
     }
-  }, [config?.api_url, config?.token]);
+  }, [config]);
 
   const generateOpenFormsData = (params) => {
     return new URLSearchParams({ ...params }).toString();
   };
-
+  if (isLoading || !config?.api_url || !config?.token) {
```

Comment:
Wouldn't returning `null` on `isLoading` here cause the spinner to never show when loading in the <Combobox ..>